integrations/flink_connector/flink-connector-timestream/src/main/java/com/amazonaws/samples/connectors/timestream/DefaultWriteRequestFailureHandler.java

```diff
@@ -110,11 +110,11 @@ void handleRetryableException(final List<Record> records,
             // if it's AwsServiceException, log it along StatusCode and RequestId
             AwsServiceException se = (AwsServiceException) e;
             // no reason to log stack trace - there won't be anything what can help
-            LOG.error("Retryable '{}' occurred while inserting to Timestream. Records insertion will be retried. " +
-                    "Details: Status Code: {}, Request ID: {}", e.getClass().getSimpleName(), se.statusCode(), se.requestId());
+            LOG.error(String.format("Retryable '%s' occurred while inserting to Timestream. Records insertion will be retried. " +
+                    "Details: Status Code: %s, Request ID: %s", e.getClass().getSimpleName(), se.statusCode(), se.requestId()), e);
```

Comment:
Just print `e` directly? There is no `e` content in the log string.

Reply:
If I will print e.toString we will loose stacktrace, based on my experience the format that I use (String, Throwwable), is most informative, but if we need just simple original error message, I can modify the code